release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -3746,9 +3746,103 @@ Space precluded documenting all of the bug fixes for this release in
+include the forward plug-in case. As a result, the correct output number then
+the correct file buffer size is now derived when the forward plug-in is
+configured.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1569544[*BZ#1569544*])
+
+* The default write operation for fluentd to Elasticsearch is index. Writes can
+trigger unnecessary `delete` operations for Elasticsearch causing extra load
+that affects performance. This bug fix utilizes the `create` operation which
+will avoid 'delete' when duplicates are introduced.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1568415[*BZ#1568415*])
+
+* Due to incorrect management of OVS flows, if two nodes rebooted and swapped IP
+addresses when they came back up, then other nodes might become unable to send
+traffic to pods on one or both of those nodes. With this bug fix, the code that
+manages OVS flows is now more careful to make the correct changes in cases of
+node IP reassignment. As a result, pod-to-pod traffic should continue to work
+correctly even after nodes swap IP addresses.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1570396[*BZ#1570396*])
+
+* Updating egress policy required blocking outgoing traffic, patching OVS flows,
+and then re-enabling traffic. However, the OVS flow generation for DNS names was
+slow, resulting in a few seconds egress traffic downtime, which may not be
+acceptable. This bug fix changes update egress policy handling to pre-populate
+all new OVS flows before blocking the outgoing traffic. As a result, the
+downtime during egress policy updates is now reduced.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1571433[*BZ#1571433*])
+
+* When using per-namespace static egress IPs, all external traffic is routed
+through the egress IP. "External" means all traffic which is not directed to
+another pod, and so includes traffic from the pod to the pod's node. When pods
+are told to use the node's IP address for DNS, and the pod is using a static
+egress IP, then DNS traffic will be routed to the egress node first, and then
+back to the original node, which might be configured to not accept DNS requests
+from other hosts, causing the pod to be unable to resolve DNS. With this bug
+fix, pod-to-node DNS requests now bypass the egress IP and go directly to the
+node, and as a result DNS now works as expected.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1570400[*BZ#1570400*])
+
+* This update fixes an issue where a node can stop reporting status if the
+connection to the Azure API is terminated uncleanly, resulting in a long timeout
+before the connection is re-established, blocking the status update loop.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1573122[*BZ#1573122*])
+
+* Resources consumed by pods whose node is unreachable or have past their
+termination grace period are no longer counted against quota.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1455743[*BZ#1455743*])
+
+[[ocp-3-7-46-bug-fixes]]
```

Comment:
@adellape - duplicate id. Fixed via: https://github.com/openshift/openshift-docs/commit/c0ea45c212c168802f3da75f669057544dfe64a4. Please make sure to include this when you update this file in master, as per: https://github.com/openshift/openshift-docs/pull/9462#issuecomment-390849122